packages/w3c-webdriver/test/14-element-interaction.test.ts

```diff
@@ -16,44 +16,43 @@ describe('Element Interaction', () => {
     });
   });
 
-  describe('sendKeys method', () => {
-    it('simulates typing in element', async () => {
+  describe('clear method', () => {
```

Comment:
I have not changed this. Just changed the order to be the same as in specification